markup_fmt/tests/fmt/jinja/interpolation/fixture.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: markup_fmt/tests/fmt.rs
+snapshot_kind: text
```

Comment:
I'm curious about how this line was introduced.

Reply:
I'm not so sure honestly, I'll investigate

Reply:
it seems to be because i'm running a too recent cargo insta version which added support for binary snapshots and thus add this to disambiguate. I've downgraded my setup and remove the line for now